pages/settings/index.tsx

```diff
@@ -444,6 +444,7 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
   );
 
   if (!session?.user?.id) {
+    Fathom.trackGoal('sign-up-complete', 0);
```

Comment:
Same issue as before since this doesn't confirm the user is new.

I think the only way to track this is during the initial account setup phase.

Reply:
Where is that in the code? the onSubmit method?